cli/scripts/e2e/e2e.ts

```diff
@@ -561,11 +561,11 @@ function runSmokeTests<T>(
     }
   );
 
-  if (["yarn", "pnpm6", "pnpm", "berry"].includes(npmClient)) {
+  if (["npm", "yarn", "pnpm6", "pnpm", "berry"].includes(npmClient)) {
     // Test `turbo prune --scope=a`
     // @todo refactor with other package managers
-    const installArgs =
-      npmClient === "berry" ? ["--immutable"] : ["--frozen-lockfile"];
+    const [installCmd, ...installArgs] =
```

Comment:
could change return value to match `{ installCmd, installArgs }`, but not a big deal